Documentation/installation/k8s-install-helm.rst

```diff
@@ -85,21 +85,19 @@ Install Cilium
 
        .. include:: ../installation/requirements-aks.rst
 
+       .. note::
+            Installing Cilium via helm is supported only for AKS BYOCNI cluster and
+            not for Azure CNI Powered by Cilium clusters.
+
```

Comment:
This is a separate change, please move it to a separate commit.

Reply:
got it